src/env-inl.h

```diff
@@ -301,6 +313,46 @@ inline int64_t Environment::get_async_wrap_uid() {
   return ++async_wrap_uid_;
 }
 
+inline void Environment::add_destroy_id(int64_t id) {
+  destroy_ids_current_->ids[destroy_ids_current_->length++] = id;
+  destroy_ids_length_++;
+  if (destroy_ids_current_->length < arraysize(destroy_ids_current_->ids))
+    return;
+  node_destroy_ids_list* old_list = destroy_ids_current_;
+  destroy_ids_current_ = new node_destroy_ids_list{ .prev = old_list };
+  destroy_ids_current_->ids[0] = id;
```

Comment:
Doesn’t this set `id` twice, once in the new instance and once at the end of the `prev` one?